pkg/rpc/client/client.go

```diff
@@ -26,8 +27,25 @@ var SuperSet = wire.NewSet(
 type RemoteURL string
 
 // NewProtobufClient is the factory method to return RPC scanner
-func NewProtobufClient(remoteURL RemoteURL) rpc.Scanner {
-	return rpc.NewScannerProtobufClient(string(remoteURL), &http.Client{})
+func NewProtobufClient(remoteURL RemoteURL) (rpc.Scanner, error) {
+	var err error
+	httpsInsecure := false
+	value, exists := os.LookupEnv("TRIVY_INSECURE")
+	if exists && value != "" {
+		httpsInsecure, err = strconv.ParseBool(value)
+		if err != nil {
+			return nil, xerrors.Errorf("unable to parse environment variable TRIVY_INSECURE: %w", err)
+		}
+	}
+
+	httpTransport := http.DefaultTransport.(*http.Transport).Clone()
+	httpTransport.TLSClientConfig.InsecureSkipVerify = httpsInsecure
```

Comment:
What if we define a new http transport?

```suggestion
	httpTransport := &http.Transport{
	        TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
	}
```